modules/storage-multi-attach-error.adoc

```diff
@@ -30,7 +30,7 @@ For most storage solutions, you can use ReadWriteMany (RWX) volumes to prevent m
 +
 For storage that does not support RWX, such as VMware vSphere, RWO volumes must be used instead. However, RWO volumes cannot be mounted on multiple nodes.
 +
-If you encounter a multi-attach error message with an RWO volume, force delete the pod on a shut down or crashed node.
+If you encounter a multi-attach error message with an RWO volume, force delete the pod on a shutdown or crashed node to avoid data loss.
```

Comment:
ISG: "Write as shown when used as a noun or an adjective - one word."

Reply:
You could provide a brief explanation about why this could cause data loss.

Reply:
Good suggestion. I've added some more detail, thanks!